styles.css

```diff
@@ -0,0 +1,3 @@
+.setting-item.search-provider .setting-item-control input {
```

Comment:
This should be moved to styles.scss.

Also a few updates to make it look not textbox-y which is a confusing UI.

```css
.setting-item.search-provider .setting-item-control input {
	width: 100%;
	border: none;
	background: none;
	outline: none;
	box-shadow: none;
	text-align: right;
}
```